pkg/driver/command_driver.go

```diff
@@ -71,7 +72,26 @@ func (d *CommandDriver) exec(op *driver.Operation) error {
 	}
 	cmd.Env = pairs
 	cmd.Stdin = bytes.NewBuffer(data)
-	out, err := cmd.CombinedOutput()
-	fmt.Fprintln(op.Out, string(out))
-	return err
+
+	// Make stdout and stderr from driver available immediately
+
+	stdout, _ := cmd.StdoutPipe()
+	go func() {
+		io.Copy(op.Out, stdout)
+	}()
+	stderr, _ := cmd.StderrPipe()
+	go func() {
+		io.Copy(op.Out, stderr)
+	}()
+	err = cmd.Start()
+	if err != nil {
```

Comment:
This project favours the style `if err := cmd.Start; err != nil {` for cases where no output variable need leaving in scope.